README.md

```diff
@@ -20,17 +20,40 @@ file.
 * jsonschema
 * Pytest to write unit tests.
 
-To finish this configuration we need add Pytest in Python Integrated Tool follow the next steps:
-
-Go File -> Settings -> Tools -> Python Integrated Tools and in Testing field in Default test runner select pytest option.  
-
 This means all required libraries can be installed from the project path using:
 ```bash 
 pip install -r requirements.txt
 ```
+To finish this configuration we need add Pytest in Python Integrated Tool follow the next steps:
+
+Go File -> Settings -> Tools -> Python Integrated Tools and in Testing field in Default test runner select pytest option.  
+
 ## About the Project
 On the 'core' folder we intend to put generic steps and functions that can be used on any API testing. 
 We also included a generic API request class, this class can be inherited to customize request options.
+
+To configure the project to run the features is necessary access to config.json file and add the respective credentials 
+in this file in the following fields:
+```
+key:
+``` 
+This field is fill with the key provided in Trello API documentation
+
+```
+token:
+```
+This field is fill with the key provided in Trello API documentation
+
+```
+email:
+``` 
+This field is fill with a valid email used to loggin in Trello application
```

Comment:
It is a mail to test features only, not to login in trello

Reply:
done